### DISCUSSION TITLE
iOS photo App share photo to RocketChat App issue

### DISCUSSION THREAD

**Discussion by kirayamatosf** [RocketChat/Rocket.Chat#32937] [Q&A]
Hello everyone,

RocketChat is really great, and I hope this issue can be resolved. 
When sharing images from the Photos app on iOS to the RocketChat app, it gets stuck and fails to send. 

However, when sharing from the RocketChat app itself using the "+" button to select images from the photo album,
 it works fine. 
What could be causing this issue?

Thank you~

**Comment by SouparnaChatterjee** [comment]
Hey, recently a friend of mine was facing a similar problem. 

This could be related to your app version or server configuration. Please check your iOS app and Rocket.Chat server configurations and update them to their latest versions. This worked out for my friend and her issue was resolved. 

However if this problem persists, it could be related to specific DNS configurations or other network settings​. 

Hope you resolve your issue soon. :)

**Comment by kirayamatosf** [comment]
Hello, Thanks for your reply.
I upgrade from 6.9 to 6.11, it fixed my problem.
Thank you very much !
